src/css/controls/imports/skipad.less

```diff
@@ -6,7 +6,7 @@
     position: absolute;
     display: flex;
     right: 0.75em;
-    bottom: 3em;
+    bottom: 3.5em;
```

Comment:
I'd prefer migrating to using pixel values. I believe this would be `3.5x16=56`. (ref: [default em size](https://github.com/jwplayer/jwplayer/blob/a935629f36a5c40448e1aa3d9c957e80abe8259e/src/css/shared-imports/vars.less#L1)) (also: [just use pixels](https://benfrain.com/just-use-pixels/))